docs/CHANGELOG.md

```diff
@@ -2,9 +2,6 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
```

Comment:
Do we want to keep these lines in the changelog?

Reply:
Personally, I would prefer to keep it (so not remove), because it can help even ourselves + potential contributions😉

Reply:
Ops, too much deletion... will restore these lines.